sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Primitives/ServiceBusSessionMessageActions.cs

```diff
@@ -17,6 +17,8 @@ public class ServiceBusSessionMessageActions : ServiceBusMessageActions
         private readonly ProcessSessionMessageEventArgs _eventArgs;
         private readonly ServiceBusSessionReceiver _receiver;
 
+        internal bool IsReleaseSession { get; set; }
```

Comment:
nit: The semantics were unclear to me at first because of the name.   Should we consider something like `ShouldReleaseSession` or `IsSessionReleaseRequested` ?